src/schemas/admin.schema.ts

```diff
@@ -0,0 +1,13 @@
+import { prop, getModelForClass } from '@typegoose/typegoose';
+
+class AdminModel {
+  @prop({ required: true })
+  email: string;
+
+  @prop({ required: true })
+  username: string;
+
+  @prop({ required: true })
+  password: string;
+}
+export const AdminSchema = getModelForClass(AdminModel);
```

Comment:
No need to create this model because there is role for `admin` and `user` in `models/user.model.st`